moveit_core/robot_state/test/robot_state_test.cpp

```diff
@@ -215,184 +214,198 @@ TEST(LoadingAndFK, SimpleRobot)
   EXPECT_NEAR_TRACED(state.getGlobalLinkTransform("base_link").translation(), Eigen::Vector3d(5, 4, 0));
 }
 
-TEST(FK, OneRobot)
+class OneRobot : public testing::Test
```

Comment:
This makes `MODEL2` available to several tests. There is no change to the actual robot descriptions.
However, clang requires a different indentation, which makes review much harder in this case.

Reply:
This should probably be in a common file that can be recycled for tests across `moveit_core`